CanvasApi.Client/EnrollmentTerms/Models/EnrollmentTermListOptions.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using CanvasApi.Client.EnrollmentTerms.Enums;
+
+namespace CanvasApi.Client.EnrollmentTerms.Models
+{
+    internal class EnrollmentTermListOptions : IEnrollmentTermListOptions
```

Comment:
The dependant class build is failing - the error looks like it will resolve itself after that initial error is fixed.